product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx

```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
 ```
 
 After assembling the command with the required images, 
-see [Installation](/postgres_distributed_for_kubernetes/latest/installation_upgrade/) for instructions on how to add the repository, 
+see [Installation](../installation_upgrade) for instructions on how to add the repository, 
 deploy the images, and create a certificate issuer. 
 
-## Example 
+For more information about how to assemble your command, see the examples: 
 
-The example command: 
+## Examples 
 
-* Pulls images from the `k8s_enterprise_pgd` repository. 
-* Uses EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
-* Uses PGD 5.4.1 as the Postgres Distributed version.
-* Uses 5.4.0 as the PGD Proxy version.
+These example commands: 
+
+* Pull images from the `k8s_enterprise_pgd` repository. 
+* Use EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
+* Use PGD 5.4.1 as the Postgres Distributed version.
+* Use 5.4.0 as the PGD Proxy version.
+
+### Example 1: Setting the environment variables before you run the deployment command
+
+Set the environment variables for the deployment command: 
+
+1.  Set the environment variable to pull images from the `k8s_enterprise_pgd` repository: 
+
+    ```
+    export REPOSITORY_NAME=k8s_enterprise_pgd
+    ```
+
+1.  Set the environment variable to use [your personal token](private_registries/#how-to-retrieve-the-token). In this example, the token is not real:
+
+    ```
+    export EDB_SUBSCRIPTION_TOKEN=aaAa1bbbBBcccccC1a1111bB1C1aBbB
+    ```
+
+1.  Set the environment variable to use EBD Postgres Advanced Server 15.6.2 as the Postgres option, 
+    and PGD 5.4.1 as the Postgres Distributed version: 
+
+    ```
+    export OPERAND_NAME=edb-postgres-advanced-pgd:15.6.2-5.4.1-1
+    ```
+
+1.  Set the environment variable to use 5.4.0 as the PGD Proxy version:
+
+    ```
+    export PGD-PROXY=edb-pgd-proxy:5.4.0
+    ```
+
+1.  Run the deployment command:
+
+    ```
+    helm upgrade --dependency-update \
+      --install edb-pg4k-pgd \ 
+      --namespace pgd-operator-system \
+      --create-namespace \
+      edb/edb-postgres-distributed-for-kubernetes \
+      --set edb-postgres-for-kubernetes.enabled=true \
+      --set image.repository=docker.enterprisedb.com/${REPOSITORY_NAME}/pg4k-pgd \ 
+      --set edb-postgres-for-kubernetes.image.repository=docker.enterprisedb.com/${REPOSITORY_NAME}/edb-postgres-for-kubernetes \
+      --set config.data.PGD_IMAGE_NAME=docker.enterprisedb.com/${REPOSITORY_NAME}/${OPERAND_NAME} \
+      --set config.data.PGD_PROXY_IMAGE_NAME=docker.enterprisedb.com/${REPOSITORY_NAME}/${PGD-PROXY}
+      --set image.imageCredentials.username=${REPOSITORY_NAME} \
+      --set image.imageCredentials.password=${EDB_SUBSCRIPTION_TOKEN}
+    ``` 
+
+### Example 2: Replacing the placeholders manually
 
 !!! Note Token 
-    Before running the command, replace the token placeholder.
+    Before running the command, replace the `EDB_SUBSCRIPTION_TOKEN` placeholder.
```

Comment:
in the previous example, you set the EDB_SUBSCRIPTION_TOKEN as an env variable.
I'd be consistent in the approach.